artifact-version-diff/src/main/java/io/quarkus/qe/Artifact.java

```diff
@@ -47,14 +50,19 @@ public String locationForVersion(String version) {
      * @param versionsTogether versions to be compared in format <upstream_version> -> <downstream_version>
      * @return false if the version are not the same
      */
-    public static boolean versionComparator(String versionsTogether) {
+    public static boolean versionComparator(String artifact, String versionsTogether) {
         // separate upstream version from downstream
         String[] versions = versionsTogether.replaceAll("\\s+", "").split("->");
 
         // prod creating artifact in format major.minor.patch.redhat-NNNNN or major.minor.patch.suffix-redhat-NNNNN
         // Change from `-` to `.` is happening only when the number is before `-`
         String version = versions[0].replaceFirst("(?<=\\d)-", ".");
 
+        // check if version is know to differ and strip it of know suffixes
+        if (isAlwaysDifferent(artifact)) {
+            version = stripArtifactVersionOfKnowSuffixes(version);
```

Comment:
stripAlphaSuffix

Reply:
Thanks, forget this. Fixed and also jenkins job rerun and it's work.